.kitchen.yml

```diff
@@ -292,7 +148,6 @@ suites:
         cfn_ephemeral_dir: <%= ENV['CFN_EPHEMERAL_DIR'] %>
         cfn_shared_dir: <%= ENV['CFN_SHARED_DIR'] %>
         cfn_cluster_user: <%= ENV['CFN_CLUSTER_USER'] %>
-        cfn_sqs_queue: <%= ENV['CFN_SQS_QUEUE'] %>
```

Comment:
Why this line was kept in this commit https://github.com/aws/aws-parallelcluster-cookbook/commit/40c715ab0f77d1d298e3fcb85f1088f21d5b64a5